src/uwtools/tests/drivers/test_fv3.py

```diff
@@ -217,371 +216,57 @@ def test_FV3_runscript(fv3obj):
+    assert "test $? -eq 0 && touch %s/done" % driverobj._rundir
 
 
-def test_FV3__run_via_batch_submission(fv3obj):
-    runscript = fv3obj._runscript_path
-    with patch.object(fv3obj, "provisioned_run_directory") as prd:
+def test_FV3__run_via_batch_submission(driverobj):
+    runscript = driverobj._runscript_path
+    with patch.object(driverobj, "provisioned_run_directory") as prd:
         with patch.object(fv3.FV3, "_scheduler", new_callable=PropertyMock) as scheduler:
-            fv3obj._run_via_batch_submission()
+            driverobj._run_via_batch_submission()
             scheduler().submit_job.assert_called_once_with(
                 runscript=runscript, submit_file=Path(f"{runscript}.submit")
             )
         prd.assert_called_once_with()
 
 
-def test_FV3__run_via_local_execution(fv3obj):
-    with patch.object(fv3obj, "provisioned_run_directory") as prd:
+def test_FV3__run_via_local_execution(driverobj):
+    with patch.object(driverobj, "provisioned_run_directory") as prd:
         with patch.object(fv3, "execute") as execute:
-            fv3obj._run_via_local_execution()
+            driverobj._run_via_local_execution()
             execute.assert_called_once_with(
-                cmd="{x} >{x}.out 2>&1".format(x=fv3obj._runscript_path),
-                cwd=fv3obj._rundir,
+                cmd="{x} >{x}.out 2>&1".format(x=driverobj._runscript_path),
+                cwd=driverobj._rundir,
                 log_output=True,
             )
         prd.assert_called_once_with()
 
 
-def test_FV3__driver_config(fv3obj):
-    assert fv3obj._driver_config == fv3obj._config["fv3"]
+def test_FV3__driver_config(driverobj):
+    assert driverobj._driver_config == driverobj._config["fv3"]
 
 
-def test_FV3__resources(fv3obj):
+def test_FV3__resources(driverobj):
     account = "me"
     scheduler = "slurm"
     walltime = "01:10:00"
-    fv3obj._driver_config["execution"].update({"batchargs": {"walltime": walltime}})
-    fv3obj._config["platform"] = {"account": account, "scheduler": scheduler}
-    assert fv3obj._resources == {
+    driverobj._driver_config["execution"].update({"batchargs": {"walltime": walltime}})
+    driverobj._config["platform"] = {"account": account, "scheduler": scheduler}
+    assert driverobj._resources == {
         "account": account,
-        "rundir": fv3obj._rundir,
+        "rundir": driverobj._rundir,
         "scheduler": scheduler,
         "walltime": walltime,
     }
 
 
-def test_FV3__runscript_path(fv3obj):
-    assert fv3obj._runscript_path == fv3obj._rundir / "runscript"
+def test_FV3__runscript_path(driverobj):
+    assert driverobj._runscript_path == driverobj._rundir / "runscript"
 
 
-def test_FV3__taskanme(fv3obj):
-    assert fv3obj._taskname("foo") == "20240201 18Z FV3 foo"
+def test_FV3__taskanme(driverobj):
+    assert driverobj._taskname("foo") == "20240201 18Z FV3 foo"
 
 
-def test_FV3__validate(fv3obj):
-    fv3obj._validate()
-
-
-# Schema fixtures
```

Comment:
All granular schema tests and their fixtures now live together in `uwtools.tests.test_schema`. They have not changed in any meaningful way, just moved.